app/Repositories/ServerRepository.php

```diff
@@ -92,6 +92,7 @@ public function create(array $data)
             'disk' => 'required|numeric|min:0',
             'service' => 'required|numeric|min:1|exists:services,id',
             'option' => 'required|numeric|min:1|exists:service_options,id',
+            'location' => 'required|numeric|min:1|exists:locations,id',
```

Comment:
Could we validate the others as well, depending on wether `auto_deploy` is set or not?